test/integ/quote-classic.test.ts

```diff
@@ -807,6 +827,62 @@ describe('quote', function () {
             }
           });
 
+          it.each(greenlistTokensPairs)(`$tokenIn.symbol -> $tokenOut.symbol`, async (tokenIn, tokenOut) => {
+            it.each([true, false])(`sendPortionEnabled = $sendPortionEnabled`, async (_) => {
+              // Arrange:
+              // - token amount from swapper
+              // - greenlist token pairs
+              // - parametrize on sendPortionEnabled
+              getAmount(1, type, tokenIn.symbol!, tokenOut.symbol!, '100');
+
+              // Act:
+              // - call classic quote
```

Comment:
I think it would be good to dedupe these tests for exactIn and exactOUt to test both?